### PR TITLE
Add ShareThrough AdServer

### PR DESCRIPTION
**Pull request by cristiandiazmrf**

### CODE REVIEW

File: json-lint/schemas/inventory/adservers/sharethrough.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                "pkey": "$1"
+            }
+        }
+    ],
+    "properties": {
+        "type": {
+            "const": "sharethrough"
+        },
+        "pkey": {
+            "type": [
```
Comment (joaquimsantmrf) [inline json-lint/schemas/inventory/adservers/sharethrough.json]: "type": "string",

**Comment by joaquimsantmrf** [comment]
retest this please

File: json-lint/schemas/inventory/adservers/sharethrough.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                "pkey": "$1"
+            }
+        }
+    ],
+    "properties": {
+        "type": {
+            "const": "sharethrough"
+        },
+        "pkey": {
+            "type":
```
Comment (ricardoribasmrf) [inline json-lint/schemas/inventory/adservers/sharethrough.json]: Indent code properly

**Comment by ricardoribasmrf** [comment]
giphy 3

**Pull request merged**